src/utils/index.js

```diff
@@ -15,6 +15,10 @@ export default {
     })
     return flag ? str : ''
   },
+  
+  isObject (val) {
+    return val !== null && typeof val === 'object' && !Array.isArray(val)
```

Comment:
Could we use `Object.prototype.toString.call(val) === '[object Object]'`?

Reply:
Maybe `val instanceof Object` ? :-)